docs/website/docs/tutorial/filesystem.md

```diff
@@ -0,0 +1,276 @@
+and default credentials. To learn more about adding credentials to your pipeline, please refer to the
+[Configuration and secrets section](../../../general-usage/credentials/complex_types#aws-credentials).
+:::
+
+## 4. Running the pipeline
+
+Let's verify that the pipeline is working as expected. Run the following command to execute the pipeline:
+
+```sh
+python filesystem_pipeline.py
+```
+
+You should see the output of the pipeline execution in the terminal. The output will also display the location of the DuckDB database file where the data is stored:
+
+```sh
+Pipeline s3_to_duckdb load step completed in 1.08 seconds
+1 load package(s) were loaded to destination duckdb and into dataset parquet_data
+The duckdb destination used duckdb:////home/user-name/quick_start/s3_to_duckdb.duckdb location to store data
+Load package 1692364844.3523829 is LOADED and contains no failed jobs
+```
+
+## 5. Exploring the data
+
+Now that the pipeline has run successfully, let's explore the data loaded into DuckDB. `dlt` comes with a built-in browser application that allows you to interact with the data. To enable it, run the following command:
+
+```sh
+pip install streamlit
+```
+
+Next, run the following command to start the data browser:
+
+```sh
+dlt pipeline s3_to_duckdb show
+```
+
+The command opens a new browser window with the data browser application. `s3_to_duckdb` is the name of the pipeline defined in the `filesystem_pipeline.py` file.
+You can explore the loaded data, run queries and see some pipeline execution details.
+
+## 6. Appending, replacing, and merging loaded data
+
+Try running the pipeline again with `python filesystem_pipeline.py`. You will notice that
+all the tables have data duplicated. This happens because by default, `dlt` appends the data to the destination table. It is very useful, for example, when you have daily data updates and you want to ingest them. In `dlt` you can control how the data is loaded into the destination table by setting the `write_disposition` parameter in the resource configuration. The possible values are:
+- `append`: Appends the data to the destination table. This is the default.
+- `replace`: Replaces the data in the destination table with the new data.
+- `merge`: Merges the new data with the existing data in the destination table based on the primary key.
+
+To specify the `write_disposition` you can set it in `pipeline.run` command:
+
+```py
+import dlt
+from dlt.sources.filesystem import filesystem, read_parquet
+
+parquet_resource = filesystem(file_glob="**/*.parquet") | read_parquet()
+pipeline = dlt.pipeline(pipeline_name="s3_to_duckdb", dataset_name="parquet_data", destination="duckdb")
+info = pipeline.run(parquet_resource, write_disposition="replace")
+print(info)
+```
+
+You can learn more about write_disposition in the [Write dispositions section](../general-usage/incremental-loading#the-3-write-dispositions).
```

Comment:
Ok so in Anton's tutorial he added examples for each of the three options - do we not want to do that here?